packages/cli/src/utils/generate.ts

```diff
@@ -437,6 +437,39 @@ function validateAndInstallMissingDependencies(basePath: string) {
   })
 }
 
+// TODO: Read the value from an environment variable
+const ENABLE_REDIRECTS_MIDDLEWARE = false
+
+// Enable redirects middleware by renaming the file from middleware__DISABLED.ts to middleware.tsß
+function enableRedirectsMiddleware(basePath: string) {
+  if (!ENABLE_REDIRECTS_MIDDLEWARE) {
+    return
+  }
+
+  try {
+    const { tmpDir } = withBasePath(basePath)
+
+    const disabledMiddlewarePath = path.join(
+      tmpDir,
+      'src',
+      'middleware__DISABLED.ts'
+    )
+
+    /* Rename the file to enable middleware functionality and then remove the disabled middleware file */
+    if (existsSync(disabledMiddlewarePath)) {
+      const enabledMiddlewarePath = path.join(tmpDir, 'src', 'middleware.ts')
+      copyFileSync(disabledMiddlewarePath, enabledMiddlewarePath)
+      removeSync(disabledMiddlewarePath)
+
+      logger.log(
+        `${chalk.green('success')} Redirects middleware has been enabled`
+      )
+    }
+  } catch (error) {
+    logger.error(error)
```

Comment:
For accounts that require the redirects feature, not creating the file correctly will result in the store not behaving correctly, right? 
If so, logging is not enough in this case, catching an error here should stop the build.

Reply:
It makes sense, I will exit the process.